tree-construction/webkit01.dat

```diff
@@ -32,8 +32,9 @@ Test
 #data
 <di
 #errors
-(1,3): eof-in-tag-name
 (1,3): expected-doctype-but-got-eof
+#new-errors
```

Comment:
Don't remove old errors yet. They could be still used by implementers. We'll remove them all in one take once we have tree construction stage errors done.